grpclb/src/test/java/io/grpc/grpclb/GrpclbLoadBalancer2Test.java

```diff
@@ -395,7 +394,6 @@ public void grpclbThenNameResolutionFails() {
     verify(helper).createOobChannel(eq(grpclbResolutionList.get(0).toEquivalentAddressGroup()),
         eq(lbAuthority(0)));
     assertEquals(1, fakeOobChannels.size());
-    ManagedChannel oobChannel = fakeOobChannels.poll();
```

Comment:
Please instead make changes around line 408 so that it looks like:
```java
assertSame(error, errorPicker.result.getStatus());
assertFalse(oobChannel.isShutdown());
```